resource/ui/ts/models/node_status.ts

```diff
@@ -141,10 +139,70 @@ module Models {
                         m("tr", [m("td", "Ranges:"), m("td", node.range_count)]),
                         m("tr", [m("td", "Leader Ranges:"), m("td", node.leader_range_count)]),
                         m("tr", [m("td", "Available Ranges:"), m("td", node.available_range_count)]),
-                        m("tr", [m("td", "Availablility:"), m("td", this._availability(nodeId))]),
+                        m("tr", [m("td", "Availablility:"), m("td", Nodes._availability(node))]),
```

Comment:
Agreed, self.class would be great.
And even if we were in a static function, the this doesn't seem to give you access to the other static elements, it's just a function representing the class.  Very strange.  I tried chaining things off of it, and it wouldn't let me in the debugger.